packages/monitor-v2/src/monitor-polymarket/common.ts

```diff
@@ -31,6 +31,21 @@ export const ONE_SCALED = ethers.utils.parseUnits("1", 18);
 
 export const POLYGON_BLOCKS_PER_HOUR = 1800;
 
+// Polymarket initializer whitelist for checking 3rd party proposals
+const POLYMARKET_INITIALIZER_WHITELIST = [
```

Comment:
Might be easier to maintain if `POLYMARKET_INITIALIZER_WHITELIST` was passed from env. That would though require allowing it to be null when not provided and not increment the `criteriaCount`.

Reply:
Yep, agreed. Done.